DBG: fix saving CPUDump and mmap dump to file

rmbreak opened the pull request:
fixes #1778 

`stringformatinline(argv[1])` was replacing a file path with \\n with a newline. For example, C:\Users\Desktop\Me\nogood.bin would cause `name = "C:\\Users\\Desktop\\Me\nogood.bin"`.

mrexodia commented:
I think a better fix would be to actually remove the (totally buggy) `StringUtils::ReplaceAll(format, "\\n", "\n")` from the string formatting functions because it's not documented that it can do that anyway 😄

rmbreak commented:
Ah, that would have been worse than where we started :p. Just added back `stringformatinline(argv[1])` and removed the `ReplaceAll` in both string formatting functions.

mrexodia commented:
Cheers!
